src/main/java/dev/mlml/Utils.java

```diff
@@ -12,4 +23,29 @@ public static boolean stringIsUserMention(String input) {
     public static boolean stringIsChannelMention(String input) {
         return input.matches("^<#\\d{17,19}>$");
     }
+
+    public static DataObject sendGetRequest(String urlString) {
```

Comment:
For completeness sake, a post request should be implemented at this point, I'll make a ticket for it as it's out of scope for this PR